modules/prebidServerBidAdapter/index.js

```diff
@@ -861,6 +861,54 @@ Object.assign(ORTB2.prototype, {
       request.ext.prebid = mergeDeep(request.ext.prebid, s2sConfig.extPrebid);
     }
 
+    // get reference to pbs config schain bidder names (if any exist)
+    let pbsSchainBidderNamesArr = [];
+    if (request.ext.prebid.schains) {
+      for (let i = 0; i < request.ext.prebid.schains.length; i++) {
+        pbsSchainBidderNamesArr = pbsSchainBidderNamesArr.concat(request.ext.prebid.schains[i].bidders);
+      }
+    }
+
+    // get reference to pbjs bidder specific schain values (if any exist)
+    let pbjsSchainValuesArr = [];
+
+    // iterate over pbjs bidder specific schains (if any exist)
+    for (let i = 0; i < bidRequests.length; i++) {
```

Comment:
Thanks for the review @dgirardi.  Good point, I agree that it would make more sense to update the request itself instead.  I went back and modified my PR, I like the route you took with your suggestion above and integrated it into my PR.  Let me know if you think any other changes should be made or if the logic seems good.